### PR TITLE
Readded ticks to primary y-axis

### PR DESCRIPTION
**Pull request by djmiley**

### CODE REVIEW

File: src/assets/js/chart/primaryChart.js [hunk 1]
```diff
@@ -134,13 +135,13 @@
...
-                            if (d === closePrice) {
-                                return d3.svg.area()(calculateCloseAxisTagPath(yAxisWidth, 14));
-                            }
+                    s.filter(function(d) { return d === closePrice; })
+                        .each(function(d) {
+                            var selection = d3.select(this);
+                            selection.classed('closeLine', true);
+                            selection.select('path').attr('d', function(d) {
+                                    return d3.svg.area()(calculateCloseAxisTagPath(yAxisWidth, 14));
+                                });
```
Comment (djmiley) [inline src/assets/js/chart/primaryChart.js]: Sounds silly, is this right indentation?

Comment (jleft) [inline src/assets/js/chart/primaryChart.js]: Looks fine to me :)

Do we need the `each` and can we chain the methods on the selection?

```js
s.filter(function(d) { return d === closePrice; })
    .classed('closeLine', true);
    .select('path')
    .attr('d', function(d) {
        return d3.svg.area()(calculateCloseAxisTagPath(yAxisWidth, 14));
    });
```

Comment (djmiley) [inline src/assets/js/chart/primaryChart.js]: Yes, much nicer. Thanks.

File: src/assets/js/chart/primaryChart.js [hunk 2]
```diff
@@ -134,14 +135,12 @@
...
-                        .select('path').attr('d', function(d) {
-                            if (d === closePrice) {
+                    s.filter(function(d) { return d === closePrice; })
+                        .classed('closeLine', true)
+                        .select('path')
+                        .attr('d', function(d) {
                                 return d3.svg.area()(calculateCloseAxisTagPath(yAxisWidth, 14));
-                            }
-                        });
+                            });
```
Comment (jleft) [inline src/assets/js/chart/primaryChart.js]: Drop indentation a level for this and the line above

**Pull request merged**